schemas/proto/foxglove/CameraCalibration.proto

```diff
@@ -67,4 +67,7 @@ message CameraCalibration {
   // 
   // This holds for both images of a stereo pair.
   repeated double P = 8; // length 12
+
+  // Frame of reference for the camera. The origin of the frame is the optical center of the camera. +x points to the right in the image, +y points down, and +z points into the plane of the image.
+  string frame_id = 9;
```

Comment:
Can we make this field first in the message (still id 9?) is it useful to do that?